### PR TITLE
Enable forum tabs in NeurIPS tests

### PR DESCRIPTION
**Pull request by zbialecki**
Add config to Blind_Submission invitation to enable forum tabs.

### CODE REVIEW

**Comment by zbialecki** [comment]
I there a way we could add a review rebuttal stage to these tests? I would like to check that the forum works with authors commenting.

**Comment by melisabok** [comment]
Yes, I can add this to the PR.

File: tests/test_neurips_conference.py [hunk 1]
```diff
@@ -1515,13 +1515,33 @@ def test_deployment_stage(self, conference, client, helpers):
...
-
         ## make submissions visible to assigned commmittee
         invitation = client.get_invitation('NeurIPS.cc/2021/Conference/-/Submission')
         invitation.reply['readers'] = {
             'values-regex': '.*'
         }
         client.post_invitation(invitation)
+
+        invitation2 = client.get_invitation('NeurIPS.cc/2021/Conference/-/Blind_Submission')
+        invitation2.replyForumViews = '''[
```
Comment (melisabok) [inline tests/test_neurips_conference.py:1526]: how did you make this work? do you see the tabs in the forum page?

Comment (zbialecki) [reply tests/test_neurips_conference.py]: Thanks for fixing this. I added this configuration using the invitation editor as well so I didn't notice this wasn't working.

**Comment by melisabok** [comment]
I logged in as an author (test@mail) and I can see the 'committee discussion' that I think the author shouldn't see. When I click on the tab I see a Review note where authors is reader. 

Screen Shot 2021-06-18 at 4 32 56 PM

**Comment by melisabok** [comment]
It seems that after a note is posted, no tab is selected, is this the expected behavior? 

I think if the posted note readers match with the current tab then we should keep that tab selected otherwise switch to the 'all' tab.

**Comment by zbialecki** [comment]
It's difficult to know what tab should be selected in certain cases. For example, if the user is on Author Discussion and changes the reader filter, should we stay on the same tab, or should no tab be selected now? I think that the tab should only be selected if the filters specified by the tab match exactly with what's selected on the page.

I agree with you about posting a note - if the new note matches the selected filters they should remain in place, but if it would be hidden given the selected filters then the should reset so the posted note is visible.

**Comment by zbialecki** [comment]
These comments will be addressed by the openreview-web PR though, so I think it's OK to merge.

**Review by melisabok — APPROVED** [review]

**Pull request merged**